pkg/networkservice/chains/nsmgrproxy/server_test.go

```diff
@@ -26,13 +26,15 @@ import (
 	"github.com/networkservicemesh/api/pkg/api/networkservice/mechanisms/kernel"
 	"github.com/networkservicemesh/api/pkg/api/registry"
 	"github.com/stretchr/testify/require"
+	"go.uber.org/goleak"
 
 	"github.com/networkservicemesh/sdk/pkg/tools/sandbox"
 )
 
 func TestNSMGR_InterdomainUseCase(t *testing.T) {
```

Comment:
Funny fact, This test testing the logic of interdomain use case with real grpc NSM services and it runs in 120 ms on my machine.

Reply:
:) :) :)